io-engine/src/grpc/v0/mayastor_grpc.rs

```diff
@@ -1665,6 +1692,38 @@ impl mayastor_server::Mayastor for MayastorSvc {
         .await
     }
 
+    #[named]
+    async fn get_rebuild_history(
+        &self,
+        request: Request<RebuildHistoryRequest>,
+    ) -> GrpcResult<RebuildHistoryReply> {
+        let ctx = GrpcClientContext::new(&request, function_name!());
+        let args = request.into_inner();
+
+        self.serialized(ctx, args.uuid.clone(), false, async move {
+            trace!("{:?}", args);
+            let rx = rpc_submit::<_, _, nexus::Error>(async move {
+                let records = nexus_lookup(&args.uuid)?
+                    .rebuild_history()
+                    .unwrap()
```

Comment:
please try to never unwrap unless it's required not to return error and you're absolutely it can never happen.

Reply:
Yes I was conscious about that, but here it's ok as in the case of no records available the returned vector will be empty but valid anyway.

Reply:
Then why does `rebuild_history` return a Result at all, can simply return `&Vec<RebuildRecord>`

Reply:
That's reasonable, removed Result as return type.